packages/dataviews/src/dataform-controls/password.tsx

```diff
@@ -34,17 +37,19 @@ export default function Password< Item >( {
 				validity,
 				type: isVisible ? 'text' : 'password',
 				suffix: (
-					<Button
-						icon={ isVisible ? unseen : seen }
-						onClick={ toggleVisibility }
-						size="small"
-						variant="tertiary"
-						aria-label={
-							isVisible
-								? __( 'Hide password' )
-								: __( 'Show password' )
-						}
-					/>
+					<InputControlSuffixWrapper variant="control">
+						<Button
+							icon={ isVisible ? unseen : seen }
+							onClick={ toggleVisibility }
+							size="small"
+							variant="tertiary"
+							aria-label={
```

Comment:
We could go with `label` directly.

```suggestion
							label={
```